.eslintrc.json

```diff
@@ -237,7 +237,8 @@
         "flowtype/sort-keys": "error",
         "babel/semi": "error",
         "jsx-a11y/no-static-element-interactions": "error",
-        "react/button-has-type": "error"
+        "react/button-has-type": "error",
+        "testing-library/no-node-access": "off"
```

Comment:
the `testing-library/no-node-access` rule prevents accessing properties of html nodes (eg. `node.parentElement`):
https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-node-access.md

this might make sense in new projects as it forces you to add things like aria labels. but it makes things hard for us when migrating tests from enzyme to rtl. i think we should disable the rule to make the migration as easy as possible.